daq/session_server.py

```diff
@@ -105,9 +106,16 @@ def _init_session(self, request):
             LOGGER.info('New session stream for %s %s/%s',
                         device_mac, request.device_vlan, request.assigned_vlan)
             assert device_mac not in self._clients, 'already registered %s' % device_mac
+            vni_set = self._vni_allocations.setdefault(request.endpoint.ip, set())
+            # vni should be unique across devices in a given remote ip.
+            vni = 1
+            while vni in vni_set:
+                vni += 1
```

Comment:
is there anything that prevent this from getting too large?

Reply:
Same here.